UNRELEASED_CHANGELOG.md

```diff
@@ -80,6 +80,32 @@
 ### ✅ Added
 - Added `MessageListView::setDeletedMessageListItemPredicate` function. It's responsible for adjusting visibility of the deleted `MessageListItem.MessageItem` elements.
 
+- Positions of messages are now a mutableList.
```

Comment:
We didn't change `positions` to mutableList and I think the next example is also outdated. Let's update it before merging